src/sirconsole.c

```diff
@@ -29,9 +29,9 @@
 #if !defined(__WIN__)
 bool _sir_write_stdio(FILE* stream, const char* message) {
     if (EOF == fputs(message, stream)) {
-        _sir_handleerr(errno);
+        _sir_handleerr(errno); // GCOVR_EXCL_START
```

Comment:
this may be a legitimate use case, because that code will also (99.9999%) never run